addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -129,9 +129,9 @@ var Wysiwyg = Widget.extend({
                 snippetMenuElement: $mainSidebar[0],
                 snippetManipulators: $snippetManipulators[0],
                 customCommands: {
-                    openMedia: { handler: this.openMediaDialog.bind(this) },
-                    openLinkDialog: { handler: this.openLinkDialog.bind(this) },
-                    saveOdoo: { handler: this.saveToServer.bind(this) }
+                    openMedia: {handler: this.openMediaDialog.bind(this)},
+                    openLinkDialog: {handler: this.openLinkDialog.bind(this)},
+                    saveOdoo: {handler: this.saveToServer.bind(this)}
```

Comment:
The previous verion was correct IMO.

Reply:
Uh that's strange indeed. Now `prettier` saves it automatically like this and I can't save the old version so maybe you could do this for me while I figure out the configuration issue?

Reply:
Actually you may be right. There seems to be something weird with our linter. I thought it was supposed to only apply to ts files but I just noticed it is warning me for odoo js files as well.